src/legacy/core_plugins/kibana/public/home/components/tutorial_directory.js

```diff
@@ -45,32 +44,59 @@ const ALL_TAB_ID = 'all';
 const SAMPLE_DATA_TAB_ID = 'sampleData';
 
 const homeTitle = i18n.translate('kbn.home.breadcrumbs.homeTitle', { defaultMessage: 'Home' });
-const addDataTitle = i18n.translate('kbn.home.breadcrumbs.addDataTitle', { defaultMessage: 'Add data' });
+const addDataTitle = i18n.translate('kbn.home.breadcrumbs.addDataTitle', {
+  defaultMessage: 'Add data',
+});
 
 class TutorialDirectoryUi extends React.Component {
-
   constructor(props) {
     super(props);
 
-    this.tabs = [{
-      id: ALL_TAB_ID,
-      name: this.props.intl.formatMessage({ id: 'kbn.home.tutorial.tabs.allTitle', defaultMessage: 'All' }),
-    }, {
-      id: 'logging',
-      name: this.props.intl.formatMessage({ id: 'kbn.home.tutorial.tabs.loggingTitle', defaultMessage: 'Logging' }),
-    }, {
-      id: 'metrics',
-      name: this.props.intl.formatMessage({ id: 'kbn.home.tutorial.tabs.metricsTitle', defaultMessage: 'Metrics' }),
-    }, {
-      id: 'security',
-      name: this.props.intl.formatMessage({ id: 'kbn.home.tutorial.tabs.securityAnalyticsTitle', defaultMessage: 'Security analytics' }),
-    }, {
-      id: SAMPLE_DATA_TAB_ID,
-      name: this.props.intl.formatMessage({ id: 'kbn.home.tutorial.tabs.sampleDataTitle', defaultMessage: 'Sample data' }),
-    }];
+    this.tabs = [
+      {
+        id: ALL_TAB_ID,
+        name: this.props.intl.formatMessage({
+          id: 'kbn.home.tutorial.tabs.allTitle',
+          defaultMessage: 'All',
+        }),
+      },
+      {
+        id: 'logging',
+        name: this.props.intl.formatMessage({
+          id: 'kbn.home.tutorial.tabs.loggingTitle',
+          defaultMessage: 'Logging',
+        }),
+      },
+      {
+        id: 'metrics',
+        name: this.props.intl.formatMessage({
+          id: 'kbn.home.tutorial.tabs.metricsTitle',
+          defaultMessage: 'Metrics',
+        }),
+      },
+      {
+        id: 'siem',
+        name: this.props.intl.formatMessage({
+          id: 'kbn.home.tutorial.tabs.siemTitle',
+          defaultMessage: 'SIEM',
+        }),
+      },
+      {
+        id: SAMPLE_DATA_TAB_ID,
+        name: this.props.intl.formatMessage({
+          id: 'kbn.home.tutorial.tabs.sampleDataTitle',
+          defaultMessage: 'Sample data',
+        }),
+      },
+    ];
 
     let openTab = ALL_TAB_ID;
-    if (props.openTab && this.tabs.some(tab => { return tab.id === props.openTab; })) {
+    if (
+      props.openTab &&
+      this.tabs.some(tab => {
+        return tab.id === props.openTab;
+      })
+    ) {
```

Comment:
Optional: You can do this on one line like so if you want it more compact

`this.tabs.some(tab => tab.id === props.openTab)`

But if this is just a linter changing things, it's fine to leave as is. No biggie.

Reply:
Yeah, this was the linter's doing :)